.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## Description
+Provide a brief overview of what is not working.
```

Comment:
@fazerlicourice7 rather than writing ```what is not working``` it could be ```issue/enhancement```. As sometimes, people give enhancements also. I guess you will get my point. :)